Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/revert.py

```diff
@@ -42,12 +42,16 @@ class Revert(Command):
     help = 'Revert provided list of commits and create a pull-request with this revert commit'
     REVERT_TITLE_TEMPLATE = 'Unreviewed, reverting {}'
     REVERT_TITLE_RE = re.compile(r'^Unreviewed, reverting {}'.format(Commit.IDENTIFIER_RE.pattern))
-    RES = [
+    RES_RADAR = [
```

Comment:
I know this is old code, but this seems very wrong. This is duplicating regexes that are in `OpenSource/Tools/Scripts/libraries/webkitbugspy/webkitbugspy` already, tied to `Tracker` instances. So we shouldn't be re-declaring these here. Also, we shouldn't be trying to extract radars (or bugillas!) from repositories which don't have those trackers configured.